vault/src/issue.rs

```diff
@@ -201,11 +203,21 @@ async fn process_transaction_and_execute_issue<B: BitcoinCoreApi + Clone + Send
                 let proof = btc_rpc.get_proof_for(txid.clone(), &block_hash).await?;
 
                 info!("Executing issue with id {}", issue_id);
-
-                // this will error if someone else executes the issue first
-                provider
+                match provider
                     .execute_issue(issue_id, H256Le::from_bytes_le(&txid.as_hash()), proof, raw_tx)
-                    .await?;
+                    .await
+                {
+                    Ok(_) => (),
+                    Err(RuntimeError::XtError(SubxtError::Runtime(SubxtRuntimeError::Module(SubxtModuleError {
```

Comment:
Great, if I recall correctly this didn't use to be possible. If we end up doing this more specific error checking, it would be good to refactor it to something like `Err(e) if e.is_issue_completed() => ...`. But for now it's fine